.github/workflows/build-google_play.yml

```diff
@@ -0,0 +1,74 @@
+name: "[CI/CD] Build Google Play release bundle"
+
+on:
+  workflow_call:
+    inputs:
+      github_run_number:
+        required: true
+        type: string
+    secrets:
+      CR_PAT:
+        required: true
+      GH_TOKEN:
+        required: true
+      MATCH_DEPLOY_KEY:
+        required: true
+      MATCH_PASSWORD:
+        required: true
+      MATCH_REPO:
+        required: true
+      GH_PACKAGE_REGISTRY_PAT:
+        required: true
+
+
+jobs:
+  ## --- GET GAME METADATA ---
+  gather_game_data:
+    name: Gather game metadata
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout game repo
+        uses: actions/checkout@v2
+
+      - name: Checkout private actions
+        uses: actions/checkout@v2                     
+        with:
+          repository: frvraps/frvr-gh-workflows
+          token: ${{ secrets.GH_TOKEN }}
+          path: ./actions
+
+      - id: metadata
+        name: Execute metadata action
+        uses: ./actions/.github/actions/getmetadata
+    outputs:
+      game_name: ${{ steps.metadata.outputs.game_name }}
+      deploy_msg: ${{ steps.metadata.outputs.deploy_msg }}  
+
+  ## --- Generate Google Play project for game ---
+  generate_googleplay:
+    name: Generate Google Play project
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    needs: [gather_game_data]
+    container:
+      image: ghcr.io/frvraps/fst@sha256:460d5348a9294ab18e907fa8480971bc765036981ca8337079988e36eaabde99
```

Comment:
This image is a snapshot of the tools and internal and might be updated before the merge